tools/1_wara_collector.ps1

```diff
@@ -238,9 +240,9 @@ $Script:Runtime = Measure-Command -Expression {
 
   function Connect-ToAzure {
     # Connect To Azure Tenant
-    Write-Host "Authenticating to Azure"
-    if ($Script:ShellPlatform -eq 'Win32NT')
+    if ($PSVersionTable.OS -eq 'Win32NT')
```

Comment:
$PSVersionTable.OS doesn't return 'Win32NT' so this wouldn't work. I think you're looking for $PSVersionTable.Platform, but that's what $Script:ShellPlatform is already holding.

Reply:
Hey @kpoineal , good catch. I've fixed with a commit.